packages/composables/src/logic/useCartItem.ts

```diff
@@ -91,6 +94,31 @@ export function useCartItem({
 
   const isPromotion = computed(() => cartItem.type === "promotion");
 
+  const getProductQtySteps = computed(() => {
```

Comment:
for the `getProductQtySteps()` function, it looks pretty similar like the mentioned helper function. As the interface of `quantityInformation` and `product` are similar for having the same properties (`maxPurchase`, `minPurchase` and `purchaseSteps`) it's tempting to generalize the logic inside packages/helpers/src/product/getProductQtySteps.ts and use it here instead? if the types fit of course 🥲